services/database/serverless.yml

```diff
@@ -62,6 +56,7 @@ provider:
             - "s3:GetObject"
             - "s3:ListBucket"
             - "s3:PutObject"
+          Resource: "*"
```

Comment:
How did it work without this before? :)

Reply:
It broke the build so I could be wrong about some of this...

Reply:
Ok I think I figured it out. It is if there are no functions AND no IAM roles being created, then you don't need that section.

Reply:
Confirmed, that seems to be how it works.